src/com/oltpbenchmark/benchmarks/tpcc/pojo/TPCC_Metrics.java

```diff
@@ -0,0 +1,60 @@
+package com.oltpbenchmark.benchmarks.tpcc.pojo;
+
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+
+public class TPCC_Metrics {
+
+    public TestConfigurationObject TestConfiguration;
+
+    public ResultObject Results = new ResultObject();
+
+    public List<LatencyList> Latencies = new ArrayList<>();
+
+    public List<LatencyList> Failure_Latencies = new ArrayList<>();
+
+    public Map<String, List<LatencyList>> Worker_Task_Latency = new LinkedHashMap<>();
+
+    public List<RetryAttemptsObject> Retry_Attempts = new ArrayList<>();
+
+    public class LatencyList {
+        public String Transaction;
+        public int Count;
+        public double min_Latency;
+        public double Avg_Latency;
+        public double max_Latency;
+        public double P99_Latency;
+        public double Connection_Acq_Latency;
+        public double min_Conn_Acq_Latency;
+        public double max_Conn_Acq_Latency;
```

Comment:
we should use camelCase for fields in java, never underscores, and generally shouldn't use PascalCase

Reply:
I have made changes at most places, but have kept use of PascalCase, considering that this will be written in the JSON file, and should be readable.